src/patterns/page-not-found-pages/user-mistake/index.njk

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout-example.njk
+---
+
+<div class="govuk-o-width-container">
+
+  <div class="govuk-o-main-wrapper">
```

Comment:
old class names here

Reply:
Updated the layout and components classes with the new ones.